verticapy/datasets/generators.py

```diff
@@ -173,6 +218,37 @@ def gen_meshgrid(features_ranges: dict) -> vDataFrame:
     -------
     vDataFrame
         generated dataset.
+
+    Examples
+    ---------
+    .. code-block:: python
+
+        from verticapy.datasets import gen_meshgrid
+        import datetime
+
+        gen_meshgrid(features_ranges = {"name": {"type": str, "values": ["Badr", "Badr", "Raghu", "Waqas",]},
+                                        "age": {"type": int, "range": [20, 40]},
+                                        "distance": {"type": float, "range": [1000, 4000]},
+                                        "date": {"type": datetime.date, "range": ["1993-11-03", 365]},
+                                        "datetime": {"type": datetime.datetime, "range": ["1993-11-03", 365]},},)
+
+    .. ipython:: python
+        :suppress:
+
+        from verticapy.datasets import gen_meshgrid
+        import datetime
+        import verticapy as vp
+        html_file = open("datasets_data_generators_gen_meshgrid.html", "w")
```

Comment:
I have updated the confluence page. Now we need to add "figures/" before such elements. 

As per the new syntax:
html_file = open("figures/datasets_data_generators_gen_meshgrid.html", "w")

Reply:
So when I use this new approach, I get hit w/ a file not found error. The old way still works though. Any ideas?

![image](https://github.com/vertica/VerticaPy/assets/40260999/9e4010d6-b11b-48b1-9640-3f83c711f182)